services/spar/src/Spar/App.hs

```diff
@@ -421,7 +427,7 @@ verdictHandlerResultCore bindCky = \case
         -- This is the first SSO authentication, so we auto-create a user. We know the user
         -- has not been created via SCIM because then we would've ended up in the
         -- "reauthentication" branch, so we pass 'ManagedByWire'.
```

Comment:
we remove 'ManagedByWire' here, doesn't this mean this comment should be updated?

Reply:
thanks, will fix in #1755!